src/main/java/com/infernostats/Preach.java

```diff
@@ -0,0 +1,25 @@
+package com.infernostats;
+
+public class Preach implements Comparable<Preach> {
```

Comment:
Needs `@AllArgsDataConstructor`, set all properties to private, and add `@Getter` as well